src/utils/convert-utils.ts

```diff
@@ -208,6 +177,43 @@ export const getPsmToHlpToCurvePath = async (
     : null;
 };
 
+export type HlpToCurvePath = {
+  hlpToken: string;
+  hlpCurveToken: string;
+  curveToken: string;
+  pool: string;
+  factory: string;
+} | null;
```

Comment:
We should try not to do this. The type should be 
```suggestion
export type HlpToCurvePath = {
  hlpToken: string;
  hlpCurveToken: string;
  curveToken: string;
  pool: string;
  factory: string;
};
```

Reply:
done